src/azure-cli/requirements.py2.windows.txt

```diff
@@ -40,7 +40,7 @@ azure-mgmt-deploymentmanager==0.2.0
 azure-mgmt-devtestlabs==2.2.0
 azure-mgmt-dns==2.1.0
 azure-mgmt-eventgrid==2.2.0
-azure-mgmt-eventhub==2.6.0
```

Comment:
I remember you mentioning that CLI already consume the multi api event hub pacakge. In that case do we need to update the version? @Juliehzl

Reply:
Just in case there are new features in the latest version. Upgrading the python package has no effect on the existing commands because there is no change for api version.